runtime/runtime.go

```diff
@@ -74,3 +74,41 @@ type PlatformRuntime interface {
 	// Delete remove a task.
 	Delete(ctx context.Context, taskID string)
 }
+
+// SandboxOpts describes sandbox creation options
+type SandboxOpts struct {
+	// RuntimeName is a runtime name containerd should bootstrap
+	RuntimeName string
+	// RuntimeOpts are options to be provided to shim when bootstrapping
+	RuntimeOpts *types.Any
+	// Spec is the sandbox spec to use
+	Spec *types.Any
+}
+
+// SandboxRuntime is an optional extension for PlatformRuntime to manage sandboxes
+type SandboxRuntime interface {
+	PlatformRuntime
```

Comment:
With the new SandboxRuntime interface, it's natural to rename PlatformRuntime as TaskRuntime. And we may add more service runtime to SandboxRuntime, such as pod specific image service for confidential computing.